stages/org.osbuild.authconfig

```diff
@@ -1,18 +1,14 @@
 #!/usr/bin/python3
 import shutil
-import subprocess
 import sys
 
 import osbuild.api
+from osbuild.util.chroot import Chroot
 
 
 def main(tree):
-    cmd = [
-        "/usr/sbin/chroot", tree,
-        "/usr/sbin/authconfig", "--nostart", "--updateall"
-    ]
-
-    subprocess.run(cmd, check=True)
+    with Chroot(tree) as chroot:
```

Comment:
(super nitpick): I wonder if we should have a helper:
```python
from osbuild.util import chroot
...
chroot.run(tree, [...])
``` 
as a shorthand here for cases where the context manager is immediately existed(?). Internally that would just do the "with Chroot(root) as chroot: chroot.run(cmd)" but maybe I'm overthinking it to safe a single line (but we seem to have this single use of run() use-case quite often so *maybe* it's worth it).

Reply:
> but maybe I'm overthinking it to safe a single line

tiny bit, but I get it.  Let's do it in a follow-up?